compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/access_to_acquire_release.mlir

```diff
@@ -0,0 +1,129 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-amdaie-access-to-acquire-release))" --split-input-file %s | FileCheck %s
```

Comment:
It's better to include another test that has "Any" access.

Reply:
Done. Added `Any` and `None` cases